README.md

```diff
@@ -1 +1,11 @@
 # LibreJargon
+An open-source jargon demystifier.
```

Comment:
We should talk about social impacts and misinformation here (climate change, covid, etc).